recipes/opencv/run_test.sh

```diff
@@ -0,0 +1,5 @@
+g++ $RECIPE_DIR/test.cpp -I$PREFIX/include  -L$PREFIX/lib -o test
+[ $(./test) != "$PKG_VERSION" ] && exit 1
+
+[ $(python -c 'import cv2; print(cv2.__version__)') != "$PKG_VERSION" ] && exit 1
+
```

Comment:
So weird that that was happening locally - but it didn't happen on Travis or CircleCI so just ignore this...